crd/nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -68,6 +68,13 @@ type NodeNetworkConfigStatus struct {
 	NetworkContainers []NetworkContainer `json:"networkContainers,omitempty"`
 }
 
+// NetworkContainerStatus defines the status of the Network Containers
+type NetworkContainerStatus struct {
+	Timestamp       metav1.Time `json:"timestamp,omitempty"`
+	LatestStatus    string      `json:"latestStatus,omitempty"`
```

Comment:
What are the values that can go here? Actual NC processing is a combination of both State and Status, i.e `Publish` and `Pending`

Reply:
This should only be updating the Final Status during DNC-RC reconciliation if the NC update was successful or if there was an error.